src/snowcli/cli/snowpark/package/commands.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+import logging
+from pathlib import Path
+
+import typer
+
+from snowcli.cli.common.decorators import global_options
+from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
+from snowcli.cli.snowpark.package.manager import PackageManager
+from snowcli.cli.snowpark.package.utils import (
+    InAnaconda,
+    NotInAnaconda,
+    RequiresPackages,
+    NothingFound,
+    CreatedSuccessfully,
+)
+
+
+app = typer.Typer(
+    name="package",
+    context_settings=DEFAULT_CONTEXT_SETTINGS,
+    help="Manage custom Python packages for Snowpark",
+)
+log = logging.getLogger(__name__)
+
+
+@app.command("lookup")
+@global_options
+def package_lookup(
+    name: str = typer.Argument(..., help="Name of the package"),
+    install_packages: bool = typer.Option(
+        False,
+        "--yes",
+        "-y",
+        help="Install packages that are not available on the Snowflake anaconda channel",
+    ),
+    **kwargs,
+):
+    """
+    Checks if a package is available on the Snowflake anaconda channel.
+    In install_packages flag is set to True, command will check all the dependencies of the packages
+    outside snowflake channel.
+    """
+    lookup_result = PackageManager().lookup(
+        name=name, install_packages=install_packages
+    )
+    PackageManager().cleanup_after_install()
+    log.info(
+        message := PackageManager().create_lookup_message(
+            lookup_result=lookup_result, name=name
+        )
+    )
```

Comment:
Should instantiate the `PackageManager` only once?

Reply:
Actually, i decided to go for 0 times - as most methods didn`t need to be packed in an object, i turned them to functions.